telemetry/specs/INT.mdk

```diff
@@ -1219,6 +1229,10 @@ header int_egress_port_tx_util_t {
     bit<32> egress_port_tx_util;
 }
 
+header int_b_occupancy_t {
+    bit<32> q_occupancy;
```

Comment:
Please make it consistent with the (example) layout provided in line 866, which has 8b buffer id.

Reply:
Let us discuss the semantics of Queue and Buffer occupancy. I will make it consistent after the discussion.

Reply:
Added Buffer ID (8 bits) and Buffer Occupancy (24 bits)